docs/operator-manual/notifications/index.md

```diff
@@ -45,3 +45,19 @@ kubectl patch app <my-app> -n argocd -p '{"metadata": {"annotations": {"notifica
 ```
 
 Try syncing an application to get notified when the sync is completed.
+
+## Namespace based configuration
```

Comment:
We should link out to the app-in-any-namespace docs page and note that this feature really only makes sense in conjunction with apps-in-any-namespace.